genie-web/src/integTest/java/com/netflix/genie/web/jpa/services/JpaJobSearchServiceImplIntegrationTest.java

```diff
@@ -324,10 +325,24 @@ public void canGetJobStatus() throws GenieException {
      */
     @Test
     public void canGetJobRequest() throws GenieException {
+        final JobRequest job1Request = this.service.getJobRequest(JOB_1_ID);
         Assert.assertThat(
-            this.service.getJobRequest(JOB_1_ID).getCommandArgs().orElseThrow(IllegalArgumentException::new),
+            job1Request.getCommandArgs().orElseThrow(IllegalArgumentException::new),
             Matchers.is("-f query.q")
         );
+        Assert.assertThat(
```

Comment:
Why not check all of them for equality?

(currently a mix of size checking and equality for just one particular field)

Reply:
Because I was modifying to test for the bug not completely re-write the tests for everything

Reply:
Right but you check one of the criteria with:
`Matchers.is(Sets.newHashSet("genie.id:cluster1", "genie.name:h2query", "sla", "yarn"))`
but not the rest of the bunch... why not check all of them?

Reply:
eh fair. Probably laziness and the knowledge this `JobSearchService` will be deleted but I'll add the rest of them the criteria to be comprehensive.

Reply:
and split your precious commits :-)